treehouses/remote
# services: add env button (fixes #1085)

Fixes #1085
## How to Test:

1. Run `treehouses upgrade cli` on raspberry pi
2. Checkout `env-var-button` branch on Android Studio.
3. Run app and go to services.
4. Tap on a service. There should be 4 different cases to look out for.
    - A service is not installed. It should not have the "edit env" button.
    - A service is installed but not running and does not have environment variables. It should not have the "edit env" button.
        - eg. turtleblocksjs, dokuwiki
    - A service is installed but not running and does have environment variables. It **should** have the "edit env" button.
        - eg. mongodb, librespeed
    - A service is running. It should not have the "edit env" button regardless if it uses env variables or not. 
5. For services where environment variables can be changed, click on the "edit env" button and you will be able to change the variables to different values. Click "Edit" to finish.
6. Run `treehouses services <service-name> config available` in terminal to see if changes are reflected.

_Note_: You can test this multiple ways. You can select each service one by one from the overview list. Or, you can change services from the drop down list in the details tab once you have already chosen a service.

![Screenshot_20200715-164053_Treehouses Remote](https://user-images.githubusercontent.com/44847682/87610376-0ba80980-c6ba-11ea-9c04-f295c4e9c55a.jpg)
